RecoVertex/BeamSpotProducer/src/BeamFitter.cc

```diff
@@ -221,6 +221,43 @@ BeamFitter::~BeamFitter() {
   delete MyPVFitter;
 }
 
+void BeamFitter::fillDescription(edm::ParameterSetDescription &iDesc) {
+  edm::ParameterSetDescription beamFitter;
+
+  beamFitter.addUntracked<bool>("Debug");
+  beamFitter.addUntracked<edm::InputTag>("TrackCollection");
+  iDesc.addUntracked<edm::InputTag>("primaryVertex", edm::InputTag("offlinePrimaryVertices"));
```

Comment:
If the default values for these parameters are defined here, would it make sense to remove them from the constructor: https://github.com/cms-sw/cmssw/blob/706cb39da184559708db0e6e9899f49995f65eb0/RecoVertex/BeamSpotProducer/src/BeamFitter.cc#L44 ?

Reply:
It seems like the main python definition of BeamFitter is here [1] (with some alternate defitions in neighboring files) - do we want to reduce the duplication of the default parameters somewhat between the fillDescriptions and the cff?

[1] https://github.com/cms-sw/cmssw/blob/02d4198c0b6615287fd88e9a8ff650aea994412e/DQM/BeamMonitor/python/BeamMonitor_cff.py#L14

Reply:
I’d prefer to have that done in a separate pull request so we can get the bug fix in.